src/makeCalculator.js

```diff
@@ -4,7 +4,43 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    reset() {
+      this.result = 0;
+
+      return this;
+    },
+
+    add(value) {
+      return (this.result += value);
```

Comment:
[CODE STYLE] - Instead of returning the result of the assignment, you should perform the operation and then return the updated `result`. This applies to lines 17, 21, 25, and 33. The `add`, `subtract`, `multiply`, and `divide` methods should only perform the operation and not return any value.